multi-agent/lib/mas/actions/providers/claude_agent/validate_vertex_connection/validate_vertex_connection.py

```
@@ -0,0 +1,124 @@
+
+    uid = "claude_agent.validate_vertex_connection"
+    name = "validate_vertex_connection"
+    description = (
+        "Validate that the Vertex AI project and region can reach "
+        "the Anthropic Claude API"
+    )
+    action_type = ActionType.VALIDATION
+    input_schema = ValidateVertexConnectionInput
+    output_schema = ValidateVertexConnectionOutput
+    version = "1.0.0"
+    tags = {"claude_agent", "vertex_ai", "validation", "gcp"}
```

Ruff (0.15.15)

[warning] Mutable default value for class attribute

(RUF012)